webpack.common.js

```diff
@@ -77,9 +77,20 @@ module.exports = {
         ],
       },
       {
-        test: /\.(woff|woff2|tff|eot|glyph|svg)$/,
+        test: /\.(woff|woff2|tff|eot|glyph)$/,
         type: "asset/resource",
       },
+      {
+        test: /\.svg$/,
```

Comment:
is this the first time we are using custom svg assets in this codebase?

Reply:
There are some font svgs, but they're all also imported as other formats too. I'm still not super familiar with the font loading story here either since in practice the fonts we include don't make it to the client.